weeb/module.py

```diff
@@ -0,0 +1,143 @@
+import discord
+from discord.ext import commands
+from discord.abc import PrivateChannel
+import aiohttp
+from typing import Optional
+from core import utils, text, logging, i18n
+
+_ = i18n.Translator("modules/fun").translate
+
+
+class Weeb(commands.Cog):
+    """
+    Weeby commands for your own Weebiness
+
+    Includes:
+        ?sauce that will give your desired sauce if you supply it with a magic number
+    """
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @classmethod
+    def doujin_tags(cls, dic):
+        characters, parodies, tags, artists, groups, languages, categories = (
+            [],
+            [],
+            [],
+            [],
+            [],
+            [],
+            [],)
+        tags = []
+        for tag in dic["tags"]:
+            if tag["type"] == "character":
+                characters.append(tag["name"])
+            elif tag["type"] == "parody":
+                parodies.append(tag["name"])
+            elif tag["type"] == "tag":
+                tags.append(tag["name"])
+            elif tag["type"] == "artist":
+                artists.append(tag["name"])
+            elif tag["type"] == "group":
+                groups.append(tag["name"])
+            elif tag["type"] == "language":
+                languages.append(tag["name"])
+            elif tag["type"] == "category":
+                categories.append(tag["name"])
```

Comment:
Why aren't you using a dictionary for this? You can instantiate it with
```py
tags = {}
for key in ("character", "parody", "tag", ...):
    tags[key] = []
for tag in dic["tags"]:
    tags[tag].append(tag["name"])
```
(I'm not sure about the code itself or if there are caveats, I'm not familiar with the raw data. But it would be much cleaner this way.)